tests/unit/workflows/test_workflows_tasks_arxiv.py

```diff
@@ -106,12 +108,7 @@ def test_arxiv_fulltext_download_logs_on_error():
     obj = MockObj(data, extra_data, files=files)
     eng = MockEng()
 
-    assert arxiv_fulltext_download(obj, eng) is None
-
-    expected = 'Cannot retrieve PDF from arXiv for 1605.03814'
-    result = obj.log._error.getvalue()
-
-    assert expected == result
+    arxiv_fulltext_download(obj, eng)
```

Comment:
Nope. If you mark it as xfailed it means that either the test is buggy, or the tested code is buggy. Either way it's something that needs to be fixed:
https://docs.pytest.org/en/latest/skipping.html#xfail-mark-test-functions-as-expected-to-fail

Here the fact that `DownloadError` is raised is not unexpected, but it's the feature that you want to test.